composer.json

```diff
@@ -4,27 +4,46 @@
     "description": "Sitemap Plugin for Sylius",
     "license": "MIT",
     "require": {
-        "php": "^7.1",
-        "sylius/sylius": "^1.0"
+        "php": "^7.2",
+        "sylius/sylius": "^1.3"
```

Comment:
I'm in doubt whether this is needed, but it probably mainly depends on upcoming changes that we do on `2.0`. With the current set-up, it should still work there and I would propose only upping it when strictly required. We can mention in README of course the versions that are actively being tested. Later we can still make it strict. WDYT?

Reply:
Not sure if Sylius did any breaking changes between `1.0` and `1.3`, so you're probably right in figuring this out and requiring the lowest possible.